azure-pipelines.yml

```diff
@@ -67,7 +67,7 @@ stages:
 
   - stage: Mobile
     displayName: Mobile libs build
-    condition: and(succeeded('WASM'), or(eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'mm2.1'), eq(variables['Build.SourceBranchName'], 'dev')))
+    condition: and(succeeded('WASM'), or(eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'mm2.1'), eq(variables['Build.SourceBranchName'], 'zcoin-enable-cpu-consumption')))
```

Comment:
shouldn't `Build.SourceBranchName` be reverted to back to `dev` now that this PR is tested?

Reply:
Done